examples/sadtalker/README.md

```diff
@@ -0,0 +1,124 @@
+# SadTalker :sob:
+
+A Mindspore implementation of [SadTalker](https://arxiv.org/pdf/2211.12194.pdf)  based on its original implemetation.
+
+## Introduction
+SadTalker is a novel system for a stylized audio-driven single image talking head videos animation using the generated realistic 3D motion coefficients (head pose, expression) of the 3DMM.
+
+<p align="center">
+<img src="https://github.com/hqkate/sadtalker/assets/26082447/d9d3b2d5-1e80-4304-84b4-768ce2b9814c" title="SadTalke" width="50%"/>
+
+<br>
+<b>TL;DR: &nbsp;&nbsp;&nbsp;&nbsp;&nbsp; single portrait image 🙎‍♂️  &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;+  &nbsp;&nbsp;&nbsp;&nbsp;&nbsp; audio 🎤  &nbsp;&nbsp;&nbsp;&nbsp;&nbsp; =  &nbsp;&nbsp;&nbsp;&nbsp;&nbsp; talking head video 🎞.</b>
+<br>
+
+</p>
+
+
+## Installation
```

Comment:
Please supplement the MS versions and CANN versions that are tested.

Reply:
Thanks for the review, the info has been updated, please check~